.github/workflows/python_wheels.yml

```diff
@@ -225,7 +243,7 @@ jobs:
           run: |
             # Install openmpi and mpi4py by conda
             # Install conda
-            dnf install -y --nobest --setopt=install_weak_deps=False wget openssh-clients.$(uname -m)
+            dnf install -y --nobest --setopt=install_weak_deps=False wget openssh-clients
```

Comment:
Just curious ... is this fixing some pre-existing bug?

Reply:
No, I don't think so. Just seems nicer to let the package manager do it's job and find the right package for us.